ios-base/DroidKaigi 2020/Session/Models/SessionDataProvider.swift

```diff
@@ -0,0 +1,16 @@
+import ios_combined
+import RxSwift
+
+final class SessionDataProvider {
+    func fetchSessions() -> Observable<[Session]> {
+        return Observable.create { observer -> Disposable in
+            ApiComponentKt.generateDroidKaigiApi().getSessions(callback: { response in
+                let model = ResponseToModelMapperKt.toModel(response)
+                observer.onNext(model.sessions)
```

Comment:
Why don't you use `Single` instead of `Observable` because of just a HTTP get response.
Maybe, just add `onComplete` after `onNext` might be fine.

Reply:
I think so too. I will change the code to use `Single`. Thanks! 👍